mongo/integration/change_stream_test.go

```diff
@@ -586,6 +586,23 @@ func TestChangeStream_ReplicaSet(t *testing.T) {
 			assert.Equal(mt, 2, numClearedEvents, "expected two PoolCleared events, got %d", numClearedEvents)
 		})
 	})
+	mt.RunOpts("call to cursor.Next after cursor closed", mtest.NewOptions().MinServerVersion("4.0"), func(mt *mtest.T) {
+		cs, err := mt.Coll.Watch(mtest.Background, mongo.Pipeline{})
+		assert.Nil(mt, err, "Watch error: %v", err)
+		defer closeStream(cs)
+		// Generate insert events
+		generateEvents(mt, 5)
+		// Call Coll.Drop to generate drop and invalidate event
+		assert.Nil(mt, mt.Coll.Drop(mtest.Background), "expected Nil but returned %v", err)
+		// Test that all events were successful
+		for i := 0; i < 7; i++ {
+			assert.True(mt, cs.Next(mtest.Background), "expected to return True, but returned False")
+		}
+		operationType := cs.Current.Lookup("operationType").String()
```

Comment:
Use `StringValue()` instead of `String()`. That won't have the additional quotes on it, so the assertion below can use `operationType` rather than slicing off the first and last characters.

Reply:
done